mockgcp/mockdataproc/util.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mockdataproc
+
+func PtrTo[T any](t T) *T {
```

Comment:
Or using this one https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/ac5a39494d2a2fe6939b995f3d931ac2e9475beb/mockgcp/common/workflows/utils.go#L17?